src/polyfills/custom-elements.js

```diff
@@ -834,7 +834,7 @@ function subClass(Object, superClass, subClass) {
  * @param {!Function=} opt_ctor
  */
 export function install(win, opt_ctor) {
-  if (isPatched(win)) {
+  if (isPatched(win) || !win.document) {
```

Comment:
`isPatched()` calls `win.HTMLElement.toString()` so we probably should reverse the order of these clauses. Also minor readability suggestion:

```js
const shouldInstall = win.document; // Don't install in no-DOM environments e.g. worker.
if (!shouldInstall || isPatched(win)) {
```

Reply:
This is resolved in e090680 . Thanks for the suggestion.